signer/rsapss/rsapss.go

```diff
@@ -0,0 +1,165 @@
+package rsapss
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha1"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"go.mozilla.org/autograph/signer"
+)
+
+const (
+	// Type of this signer is "rsapss"
+	Type = "rsapss"
+)
+
+// RSAPSSSigner holds the configuration of the signer
+type RSAPSSSigner struct {
+	signer.Configuration
+
+	// key is the parsed RSA private key to sign hashes
+	key *rsa.PrivateKey
```

Comment:
would you like to make this hsm-proof right away? in which case, you should manipulate a crypto.PrivateKey instead of an *rsa one.